### DISCUSSION TITLE
Development environments

### DISCUSSION THREAD

**Discussion by rndquu** [pavlovcik/uad-ui#652] [Ideas]
Let's use different environments for https://uad.ubq.fi/

I mean:
1. setup https://uad.ubq.fi/ to track `ubiquity/ubiquity-dollar#main` branch for contracts and `ubiquity/ubq.fi#main` branch for UI
2. setup https://dev.uad.ubq.fi/ to track `ubiquity/ubiquity-dollar#development` branch for contracts and `ubiquity/ubq.fi#development` branch for UI
3. setup https://stage.uad.ubq.fi/ to track `ubiquity/ubiquity-dollar#release-X.X.X` branch for contracts and `ubiquity/ubq.fi#X.X.X` branch for UI. Here the `release-X.X.X` branch is the branch that is QAed right now and is about to merged to the `main` branch and deployed to production

This way we could setup different deployment scripts for ubiquity contracts depending on the environment

What do you think @ubiquity/development ?

**Comment by 0x4007** [comment]
Are you able to walk me through how to do these on Cloudflare? I believe that `1.` and `2.` are simple CNAME DNS entries, but I believe that `3.` will require some type of custom Cloudflare Worker? 

I suggest you use any domain name you own, and try setting up your idea there. Then I can copy your settings and reflect them on https://ubq.fi

**Comment by 0x4007** [comment]
I converted this into #658
